core/mondoo-okta-security.mql.yaml

```diff
@@ -1,379 +1,1985 @@
+          - uid: mondoo-okta-security-password-settings-password-sms-recovery
+          - uid: mondoo-okta-security-password-settings-password-question-recovery
 queries:
-  - uid: mondoo-okta-security-okta-mfa-access
-    title: Require MFA for all users
+  - uid: mondoo-okta-security-limit-super-admins
+    title: Limit the number of super admins
+    impact: 100
+    docs:
+      desc: |
+        Admin roles allow you to control user access to a range of Okta functions. You can assign more than one role to an individual admin if their job requires them to perform actions that span multiple roles. This role can create other admins, assign or remove permissions, and perform all other admin activities. The super admin has the highest permissions of all admin roles.
+
+        ## Okta Recommends 
+
+        Limit the number of super admins only to users who require super admin access. An org shouldn't have more than:
+
+        - 50 percent of admins have super admin privileges
+        - 15 super admins
+
+        All other admins should only have the permissions as required for their role.
+
+        Plan for a recurring assessment of all admin privileges to ensure that these best practices are met.
+    refs:
+      - url: https://help.okta.com/en-us/Content/Topics/Security/healthinsight/healthinsight-security-task-recomendations.htm  
+        title: HealthInsight tasks and recommendations
+      - url: https://help.okta.com/en-us/Content/Topics/Security/network/network-zones.htm
+        title: Network Zones 
+      - url: https://help.okta.com/en-us/Content/Topics/Security/threat-insight/configure-threatinsight.htm
+        title: Configure Okta ThreatInsight 
+      - url: https://help.okta.com/en-us/Content/Topics/Security/Administrators.htm
+        title: Administrators 
+      - url: https://help.okta.com/en-us/Content/Topics/Security/administrators-super-admin.htm
+        title: Super Administrators 
+    props:
+      - uid: maxOktaSuperAdmins
+        title: Maximum number of Okta Super Admins
+        mql: return 15
+    variants:
+      - uid: mondoo-okta-security-limit-super-admins-runtime
+  - uid: mondoo-okta-security-limit-super-admins-runtime
+    title: Limit the number of super admins
     impact: 100
+    filters: asset.platform == "okta-org"
+    mql: okta.users.where( roles.map(type).contains("SUPER_ADMIN") ).length < props.maxOktaSuperAdmins
+    docs:
+      remediation:
+        - id: console
+          desc: |
+            ### Change admin privileges to a user or an Okta group
+
+            1. In the Admin Console, go to **Security > Administrators**.
+            2. Under **Admin Roles**, select the Super filter to display only super administrators.
+            3. Next to each user entry, click **Edit**. The Edit Administrator window is displayed.
+            4. From the list of administrator roles, assign a role other than super admin to the user.
+            5. Click **Update Administrator**.
```

Comment:
```suggestion
            5. Select **Update Administrator**.
```

Reply:
@tas50 these are not our steps. they are from Okta's docs. They are copy/pasted.